src/View/mainpage/StructureTab.py

```diff
@@ -279,10 +284,15 @@ def structure_checked(self, state, roi_id):
         self.request_update_structures.emit()
 
     def save_new_rtss(self, event=None):
-        rtss_directory = str(Path(self.patient_dict_container.get("file_rtss")).parent)
-        save_filepath = save_filepath = QtWidgets.QFileDialog.getSaveFileName(self.parentWidget(), "Save file",
-                                                                              rtss_directory)[0]
-        if save_filepath != "":
-            self.patient_dict_container.get("dataset_rtss").save_as(save_filepath)
+        rtss_directory = str(Path(self.patient_dict_container.get("file_rtss")))
+
+        confirm_save = QtWidgets.QMessageBox.information(self, "Confirmation",
+                                                 "Are you sure you want to save the modified RTSTRUCT file? This will "
+                                                 "overwrite the existing file. This is not reversible.",
```

Comment:
you might want to back up the original somewhere that won't get automatically loaded in a future run, but a support person could get to (e.g. rtss.dcm -> rtss.dcm.orig, and then you overwrite rtss.dcm... and if there is already an rtss.dcm.orig, start incrementing rtss.dcm.orig.001 or don't bother overwriting again... they have the true original and incremental changes are not as important... but if this is where the RadOnc is putting all of their effort, they might not be happy to lose incremental work if they made some kind of mistake).

Reply:
I will create this into a GitHub issue.